src/lib/db/addon-store.ts

```diff
@@ -71,18 +72,19 @@ export default class AddonStore implements IAddonStore {
         stopTimer();
         // eslint-disable-next-line @typescript-eslint/naming-convention
         const { id, created_at } = rows[0];
-        return { id, createdAt: created_at, ...addon };
+        return this.rowToAddon({ id, createdAt: created_at, ...addon });
```

Comment:
Without mapping this, we could return a "created" payload with a missing description property. Using the `rowToAddon` method, the `description` property will be set to `null` if it is missing. This seems like a bug. Getting the same addon via a GET (or PUT) request would return the `description` property (as `null` if it was missing)